doc/PROFICIENCY_LIST.md

```diff
@@ -1,144 +1,144 @@
+| `prof_food_canning`           | Canning Food           | Using pressure canning or tinning to safely store food in sealed containers. | prof_preservation
+| |
+| `prof_cheesemaking_1`         | Principles of Cheesemaking | The basics of curdling and coagulating milk and similar things without the wrong kinds of bacteria and mold winning out. | None
+| `prof_cheesemaking_2`         | Cheesemaking Expert     | Taking the basics of curdled milk and making it delicious. | prof_cheesemaking_1
+| |
+| `prof_fermenting`             | Culinary Fermentation   | The basics of fermenting for human consumption, including sanitary technique, avoiding harmful molds, and identifying common problems. | None
+| `prof_brewing`                | Brewing                 | The fine art of brewing grains into beers. | prof_fermenting
+| `prof_winemaking`             | Winemaking              | The knowledge of how to turn sugary fruits, and occasionally vegetables, into fermented beverages. | prof_fermenting
+| `prof_lactofermenting`        | Lactofermentation       | The microbiology of most non-alcoholic fermented edibles, from sauerkraut to sourdough. | prof_fermenting
+| |
+| `prof_distilling`             | Distillation            | The process of separating two mixed liquids by their different boiling points. | None
+| `prof_beverage_distilling`    | Beverage Distillation   | Applying the science of distillation to alcoholic beverages to get something tasty. | prof_distilling
+| |
+| `prof_intro_chemistry`        | Principles of Chemistry | You are beginning to grasp a general idea of how elements and compounds react with each other. | None
+| `prof_intro_biology`          | Principles of Biology   | You are beginning to gain a general idea of how various living beings function. | None
+| `prof_organic_chemistry`      | Organic Chemistry       | Knowledge of the branch of chemistry that studies and uses carbon-containing compounds. | prof_intro_chemistry
+| `prof_inorganic_chemistry`    | Inorganic Chemistry     | Knowledge of the branch of chemistry that studies and uses compounds not containing carbon. | prof_intro_chemistry
+| `prof_biochemistry`           | Biochemistry            | The understanding of the chemistry of living things. | prof_organic_chemistry / prof_intro_biology
+| `prof_physiology`             | Physiology              | An in-depth understanding of how humans and animals operate. | prof_intro_biology
+| `prof_xenology`               | Xenology                | You are beginning to grasp a general idea of how alien and post-Cataclysm mutated creatures function and operate. | prof_intro_biology
+| `prof_intro_chem_synth`       | Principles of Chemical Synthesis | You're beginning to grasp the basics of chemical production. | prof_intro_chemistry
+| `prof_chem_synth`             | Complex Chemical Synthesis | The proper understanding of safe chemical synthesis and how to clean your tools used for the next batch. | prof_intro_chem_synth
```

Comment:
I'm confused here since it seems like the above |      | sections align internally and in the proficiency list that internal consistency is no longer being followed?